sdcm/cluster.py

```diff
@@ -3611,6 +3611,26 @@ def verify_node_setup(start_time):
     return wrapper
 
 
+def get_partition_keys(ks_cf: str, db_node: BaseNode, pk_name: str = 'pk', num_of_pk: int = 100) -> List[str]:
```

Comment:
Why on cluster.py and not utils or somewhere more general.

Reply:
because it uses 'BaseNode' which is found in cluster.py..
(so now cluster.py has 5482 lines and definitelly needs a general refactor task)

Reply:
well, that is not a reason, because you can pass the `db_node` parameter to that function, anywhere else in the code

Reply:
This will cause a cyclic import..

Reply:
so how about moving out 'BaseNode' out of cluster.py to some 'node.py'? Shouldn't be hard.

Reply:
That could be a good idea for a separate task.

Reply:
fixed eventually.